src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/IService1.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.ServiceModel;
 using System.Runtime.Serialization;
+using System;
```

Comment:
>using System [](start = 0, length = 12)

Nit: we usually put these in sorted order.  Don't make a new PR just for this.  Consider for future only.